### ISSUE TITLE
Filter a11y considerations / bugs

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-platform#507]
From @mbgower 

The basic issues are:
- the filter button is probably not best as a button. Or, at the least, it needs a ‘aria-haspopup’. **_Action: discuss_**
- [ ] the filter buttons don’t reveal any updates as they are activated by JAWS. Bug
- That filter delete button inside another button is problematic. **_Action: discuss_**
- [ ] I can’t actually delete the filter with JAWS (although I can with a keyboard with JAWS turned off). Bug
- After I’d thought about this a bit, I decided an easier solution for the whole tag filter info is to just make the tag buttons into toggle buttons. I  made another [recording with aria-pressed invoked ](https://ibm.box.com/s/j1wksp4ht1lsokh08xk58zcmrqrmfvjb)on a few tags

For more detail see slack thread: 

- https://ibm-studios.slack.com/archives/G01GCBCGTPV/p1649192797233509?thread_ts=1649182396.761059&cid=G01GCBCGTPV
